payjoin/src/send/mod.rs

```diff
@@ -583,6 +718,67 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn test_equal_amount_fee_contribution() -> Result<(), BoxError> {
+        let mut ctx = create_psbt_context()?;
+        let mut proposal: bitcoin::Psbt = PARSED_PAYJOIN_PROPOSAL.clone();
+
+        ctx.fee_contribution = None;
+        proposal.unsigned_tx.output.get_mut(0).unwrap().value =
```

Comment:
In the future, especially for tests, I think output[0] would work here and still panic. The only time to use get_mut(0) is if you wanted to handle it without a panic.